pkg/tasks/tasks.go

```diff
@@ -104,7 +104,15 @@ func WithHostnameOSAndProbes(t Tasks) Tasks {
 // WithFullInstall with install binaries (using WithBinariesOnly) and
 // orchestrate complete cluster init
 func WithFullInstall(t Tasks) Tasks {
-	return WithBinariesOnly(t).
+	return Tasks{
+		{
+			Fn: func(s *state.State) error {
+				s.Logger.Infoln("Disable nm-cloud-setup on aws rhel nodes...")
```

Comment:
this task will run unconditionally, which means this log message will appear even if there are no RHEL machines on AWS provider

Reply:
So best is, making it generic. Will change it to Disable 'nm-cloud-setup tool...'